h/models/token.py

```diff
@@ -45,7 +44,10 @@ def __init__(self, **kwargs):
 
     @classmethod
     def get_by_userid(cls, session, userid):
-        return session.query(cls).filter(cls.userid == userid).first()
+        return session.query(cls) \
+            .filter_by(userid=userid) \
+            .order_by(cls.created.desc()) \
+            .first()
```

Comment:
By the way I think this style might be better:

```python
return (session.query(cls)
  .filter_by(userid=userid)
  .order_by(cls.created.desc())
  .first())
```

because PEP8 says to avoid \'s for line continuation whenever possible. It's less characters (just an extra `(` and `)` instead of 3+ `\`'s), and it's easier to edit as you don't have to add a new `\` if you add a new line.